lib/game.rb

```diff
@@ -1,18 +1,27 @@
 class Game
-  attr_reader :board, :current_player, :opponent
+  attr_reader :board, :current_player, :opponent, :save_game, :exit_game
 
   def initialize(board, player1, player2)
     @board = board
     @player1 = player1
     @player2 = player2
     @current_player = @player1
     @opponent = @player2
+    @save_game = false
```

Comment:
I was wondering what is your opinion on having primitive types as statuses (`save_game` and `exit_game`). 

For example, if in the future you were requested to implement exiting game with different statuses (like you do with `0` right now), you couldn't store this information in `@exit_game` status as it's only a primitive value.